src/modules/collection/reducer.ts

```diff
@@ -261,7 +262,8 @@ export function collectionReducer(state: CollectionState = INITIAL_STATE, action
             ...state.data[collection.id],
             forumLink
           }
-        }
+        },
+        loading: loadingReducer(state.loading, action)
```

Comment:
this line is to update the loading state when creating the forum post for a collection is successful